tutorials/external-models-tribuo-v4.ipynb

```diff
@@ -133,10 +133,7 @@
     }
    ],
    "source": [
-    "var lrTrainer = new LinearSGDTrainer(new LogMulticlass(),new AdaGrad(0.1),3,30000,Trainer.DEFAULT_SEED);\n",
-    "var lrModel = lrTrainer.train(mnistTrain);\n",
-    "var lrEvaluation = labelFactory.getEvaluator().evaluate(lrModel,mnistTest);\n",
-    "lrEvaluation.toString();"
```

Comment:
this looks like a fat finger.  "b" is not code per se....

Reply:
Fixed, thanks.